lib/lib.go

```diff
@@ -129,12 +96,12 @@ func isIDN(domain string) bool {
 }
 
 // IsMatchingCert checks if certificate matches the regexp
-func IsMatchingCert(config *Configuration, result *Result, reg *regexp.Regexp) bool {
+func IsMatchingCert(homoglyphs map[string]string, result *model.Result, reg *regexp.Regexp) bool {
```

Comment:
the map is pretty big, can we pass it's pointer for avoiding copy of memory?

Reply:
Yes, it's done. We'll use a pointer to homoglyph map.